application/docker-compose.yml

```diff
@@ -7,13 +7,12 @@ services:
       - 8080:8080
     networks:
       - ehrbase-net
+    env_file:
+      - .env.ehrbase
     environment:
       DB_URL: jdbc:postgresql://ehrdb:5432/ehrbase
       DB_USER: ehrbase
       DB_PASS: ehrbase
-      AUTH_TYPE: BASIC
-      AUTH_USER: ehrbase-user
-      AUTH_PASSWORD: SuperSecretPassword
```

Comment:
Should I then create a "receipt" section to the documentation with examples on how to enable BASI and OAUTH with a auth server as keycloak? Then we can leave this configuration as it is without security to enable the users to start as quick as possible?

Reply:
I would prefer to active the basic auth here by default, so that never ever someone accidentally deploys EHRbase without any auth. An alternative would be to clearly mark all docker instructions we give as *development* info.
But a section in the docs with some config ideas sounds good, too. Their maintainability might be problematic though.